### DISCUSSION TITLE
How to plan trajectory using only Graph Planner?

### DISCUSSION THREAD

**Discussion by chenzyn** [NVlabs/curobo#203] [Software Q&A]
When I set `enable_graph=True, enable_opt=False`, the result shows that `result.graph_plan` is none.

**Comment by chenzyn** [comment]
But when I set `enable_graph=False, enable_opt=True`, the result `result.optimized_plan` is not None.

**Comment by balakumar-s** [comment]
For both cases, you can get trajectory with: `q_traj = result.get_interpolated_plan()`

**Reply by chenzyn** [reply]
When setting the parameters as `enable_graph=True` and `enable_opt=False`, the number of successful instances is 0. However, when setting `enable_graph=False` and `enable_opt=True`, the number of successful instances is normal.

**Reply by balakumar-s** [reply]
What robot are you using? Can you share a python example for us to debug?

**Comment by jih189** [comment]
result = motion_gen.plan_single_js(start_state, goal_state, MotionGenPlanConfig(
        enable_graph=True, 
        enable_opt=False,
        use_nn_ik_seed=False,
        need_graph_success=False,
        max_attempts=1,
        timeout=10.0,
        enable_finetune_trajopt=False,
        parallel_finetune=False,
        finetune_attempts=0
    )
)
end_time = time.time()
print("Time taken for planning: ", end_time - start_time)

# print all time taken
print("Time taken for graph_time: ", result.graph_time)
print("Time taken for finetune_time: ", result.finetune_time)
print("Time taken for trajopt_time: ", result.trajopt_time)
print("Number of attempts: ", result.attempts)

# check the number of valid states in graph
traj = result.get_interpolated_plan()
